rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -128,9 +130,9 @@ def __init__(self, batch_size=-1, num_threads=-1, device_id=-1, seed=1,
         self._default_cuda_stream_priority = default_cuda_stream_priority
         self._tensor_layout = tensor_layout
         self._tensor_dtype = tensor_dtype
-        self._multiplier = multiplier
+        self._multiplier = multiplier if multiplier else [1.0,1.0,1.0]
```

Comment:
I think original code should work for handling default value

Reply:
There was a codacy warning which warned about having mutable default values in init function. I changed the init function to fix the warning

Reply:
Passing the mean and std to pipeline and calculating the multiplier and offset in pipeline as discussed in the call